src/components/style.css

```diff
@@ -0,0 +1,181 @@
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 87.5rem;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
+}
+
+.bookstore {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.books {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-size: 1rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```

Comment:
Good job working on the project requirements but it would be nice if you styled your work according to the design template you were given. Please adjust the width of your nav bar and also use the correct font-style which is Roboto slab I'll add screen shorts of indicators on what you should change in your design so it best suits the original design template.

This is your design:

![Student screen](https://github.com/NalianAchei2020/bookstore/assets/79506105/fb82c93b-6af8-4113-956e-f4b0a82c15fd)

Please stick to this original design:


![Original design](https://github.com/NalianAchei2020/bookstore/assets/79506105/6730e7a1-bb97-4f23-9a5e-356cfdbfc804)

Please its important to follow design requirements, carefully inspect the template to avoid mistakes please always pay attention to detail.